tox.ini

```diff
@@ -1,11 +1,15 @@
 [tox]
-envlist = flake8
+envlist = flake8,py27,p36
```

Comment:
(absolutely optional) Space missing after comma? :P

Reply:
Sure, let me try it!

Reply:
Fixed